src/services/conseillers/repository/conseillers.repository.ts

```diff
@@ -147,25 +148,31 @@ const filtrePiecesManquantes = (piecesManquantes: boolean) => {
 };
 
 const filterIsRuptureMisesEnRelation = (
-  rupture: string,
+  ruptureOuTermine: string,
   conseillerIdsRecruter: ObjectId[],
   structureIds: ObjectId[],
   conseillerIdsRupture: ObjectId[],
+  conseillerIdsTerminerNaturelle: ObjectId[],
   piecesManquantes: boolean,
 ) => {
-  switch (rupture) {
+  switch (ruptureOuTermine) {
     case 'nouvelle_rupture':
       return {
-        statut: { $eq: rupture },
+        statut: { $eq: ruptureOuTermine },
         'conseiller.$id': { $in: conseillerIdsRecruter },
         'structure.$id': { $in: structureIds },
         ...filtrePiecesManquantes(piecesManquantes),
       };
     case 'finalisee_rupture':
       return {
-        statut: { $eq: rupture },
+        statut: { $eq: ruptureOuTermine },
         'conseiller.$id': { $in: conseillerIdsRupture },
       };
+    case 'terminee_naturelle':
+      return {
+        statut: { $eq: ruptureOuTermine },
+        'conseiller.$id': { $in: conseillerIdsTerminerNaturelle },
+      };
```

Comment:
je pense que tu devrais envoyer un statutContrat custom par exemple sans-mission et faire
```suggestion
    case 'sans-mission':
      return {
        statut: { $in: ['finalisee'_rupture, 'terminee_naturelle'] },
        'conseiller.$id': { $in: [conseillerIdsTerminerNaturelle, conseillerIdsRupture ] },
      };
```